### ISSUE TITLE
nemo-mediainfo-tab

### ISSUE THREAD

**Issue by nrroyal19** [getsolus/packages#2052]
### Please confirm there isn't an existing package request

- [X] I have searched through package requests

### Homepage

https://github.com/linux-man/nemo-mediainfo-tab

### Maintainer

- [ ] Would you like to be the maintainer for this package?

### Why should this be included in the repository?

extended nemo functionality. Almost every distro that uses nemo have it.

### Are we allowed to redistribute it?

GPL-3.0 license 

### What kind of user will use this package, and how many users do you think will use this package?

probably a nemo user will use it , anyone who like media like to use it 

### Link to source archive file

https://github.com/linux-man/nemo-mediainfo-tab/archive/refs/tags/v1.0.4.tar.gz

**Comment by EbonJaeger** [comment]
We have `nemo-audio-tab` as a part of `nemo-extensions`. Is that the same or a similar thing? The descriptions are almost the same.

**Comment by nrroyal19** [comment]
yes i have installed it but its not same , this extension create a new tab in properties that will give proper information about video and audio files.

**Comment by TraceyC77** [comment]
Unfortunately, this package hasn't seen broader interest or attracted a maintainer. Also, the software hasn't created a release since Sep 2, 2021. The repo looks dead.

Since Solus has limited resources, we aren't able to add every package that gets requested. I'm closing this request for now. If there's a new release and a maintainer steps up to take this on, we can take another look.

Thanks for your understanding.
